### PR TITLE
Added "Get Started with CNNs" example

### PR DESCRIPTION
**Pull request by Rishit-dagli**
Added a "[Getting Started with CNN](https://github.com/Rishit-dagli/tensorflow/tree/get-started-with-cnn/tensorflow/examples/get_started/get-started-with-cnn)" example under the [get_started](https://github.com/tensorflow/tensorflow/tree/master/tensorflow/examples/get_started) section. I have made sure that this example is strictly a getting started example with TensorFlow 2.x .

The example contains-

- What is a CNN?
- Loading the data
  * The Fashion MNIST dataset
- Some preprocessing
- Understanding Filters and Convolutions
- Performing simple feature extraction with filters
  * Identifying Vertical lines
- Understanding Pooling
- Implementing Convolutional layers in TensorFlow
  * Convolutions
  * Pooling
  * Accuracy and loss curves
  * Test accuracy
- Visualizing the Convolutions and Pooling
- Save the model for future use

### CODE REVIEW

**Comment by Rishit-dagli** [comment]
The commit [`f5d8597`](https://github.com/tensorflow/tensorflow/pull/40970/commits/f5d859777b3340319d44cc4f8897cad9039d0ddd) was a mistake from my end as mentioned [here](https://github.com/tensorflow/tensorflow/pull/40828#issuecomment-651910995) I messed up two branches in my TF fork. I have however fixed it.

**Comment by yashk2810** [comment]
We already have a tutorial here: https://www.tensorflow.org/tutorials/keras/classification demonstrating how to do that. Also there are many other guides/tutorials that explain.

Can you please tell why do you want to add this example?

**Comment by lamberta** [comment]
Agree with @yashk2810 
Regardless, tutorials and examples should go in either the https://github.com/tensorflow/docs or https://github.com/tensorflow/examples repo so closing this issue.

**Comment by Rishit-dagli** [comment]
> We already have a tutorial here: https://www.tensorflow.org/tutorials/keras/classification demonstrating how to do that. Also there are many other guides/tutorials that explain.
> 
> Can you please tell why do you want to add this example?

I am aware of this tutorial however this tutorial does not demonstrate using CNNs with TensorFlow. I feel this could be a good place for beginners to get started with CNNs in TF.

**Comment by Rishit-dagli** [comment]
> Agree with @yashk2810
> Regardless, tutorials and examples should go in either the https://github.com/tensorflow/docs or https://github.com/tensorflow/examples repo so closing this issue.

I feel this would be a good fit under the [get_started](https://github.com/tensorflow/tensorflow/tree/master/tensorflow/examples/get_started) section in [examples](https://github.com/tensorflow/tensorflow/tree/master/tensorflow/examples) in this repo
